controllers/src/test/scala/cromwell/pipeline/controller/RunControllerTest.scala

```diff
@@ -49,47 +49,44 @@ class RunControllerTest
         when(runService.getRunByIdAndUser(runId, accessToken.userId))
           .thenReturn(Future.failed(new RuntimeException("something went wrong")))
 
-        Get("/runs?run_id=" + runId.unwrap) ~> runController.route(accessToken) ~> check {
+        Get("/runs/" + runId.unwrap) ~> runController.route(accessToken) ~> check {
           status shouldBe StatusCodes.InternalServerError
         }
       }
 
       "returns run entity if the run id is not valid" taggedAs Controller in {
```

Comment:
```suggestion
      "returns NotFound if the run id is not valid" taggedAs Controller in {
```
This is your mistake, but please fix it while we are here

Reply:
Yep.My fault. Done.